lib/commands/context.js

```diff
@@ -380,10 +380,6 @@ extensions.connectToRemoteDebugger = async function connectToRemoteDebugger () {
 };
 
 extensions.listWebFrames = async function listWebFrames (useUrl = true) {
-  if (!this.opts.bundleId) {
```

Comment:
have you verified that having this value equal to undefined/null won't create unexpected NPE later in this method?

Reply:
Yes.
I tested with both non-app and non-bundleid case.
Some commands like get contexts, set a context and get page source worked.